docs/control-structures.rst

```diff
@@ -52,40 +57,53 @@ write::
 
 The names of output parameters can be omitted.
 The output values can also be specified using ``return`` statements.
-The ``return`` statements are also capable of returning multiple
-values, see :ref:`multi-return`.
+
+The ``return`` statements are also capable of returning multiple values using ``return (v0, v1, ..., vn)``. The number of parameters must equal the number of output parameters and they must all be the same type.
```

Comment:
Please create a new PR with just this paragraph - perhaps using `return (_a + _b, _a * _b);` as additional example code mentioned after the abstract `(v0, ...)` one.